### ISSUE TITLE
noCookie option continues to set tracking cookies

### ISSUE THREAD

**Issue by juliengiombetti** [sampotts/plyr#2503]
### Expected behaviour
The noCookie option does NOT insert any tracking cookies from youtube.

### Actual behaviour
The noCookie option continues to insert two important cookies: **YSC** and **VISITOR_INFO1_LIVE**  (which are tracking cookies). I think it's related to the call at the API for the iframe (https://www.youtube.com/iframe_api). Is there another way to do this ?

### Steps to reproduce
`const players = Array.from(document.querySelectorAll('.player')).map((p) => new Plyr(p, {
      youtube: { noCookie: true }
    }));`

`<div class="player" data-plyr-provider="youtube" data-plyr-embed-id="Cp9pk-FkE6E"></div>`

### Environment

- Browser: Chrome
- Version: 103
- Operating System: macOS
- Version: Monterey

### Link to where the bug is happening
https://codepen.io/juliengiombetti/pen/OJvRVaR

**Comment by gregogalante** [comment]
Getting same problem. Any solution?

**Comment by ffxsam** [comment]
Are you sure this is an issue? I tested on my own site, and there are no cookies being set under the youtube domain nor youtube-nocookie domain.

Also, that cookie on Codepen is a result of Codepen itself (note the cookie domain is google.com). If you open the full-page version (https://codepen.io/juliengiombetti/full/OJvRVaR), you can see there are no cookies being set:

![CleanShot 2022-12-28 at 14 19 27@2x](https://user-images.githubusercontent.com/12532733/209867815-f0738f8f-cd70-41fc-8a1a-e7947550ad1e.png)

**Comment by dr-format-d** [comment]
I agree. We are also still having this issue. If you set `noCookie: true` it will be completely ignored apparently.

![grafik](https://github.com/sampotts/plyr/assets/110388306/6d94e449-c8a5-42c2-965b-9860672a302d)
![grafik](https://github.com/sampotts/plyr/assets/110388306/3b510f9f-0ff3-4de7-abd1-f04c5a667597)
![grafik](https://github.com/sampotts/plyr/assets/110388306/d540791d-1f01-4279-977e-42cd547763d5)
